README.md

```diff
@@ -1,5 +1,10 @@
 # kafka-go [![CircleCI](https://circleci.com/gh/segmentio/kafka-go.svg?style=shield)](https://circleci.com/gh/segmentio/kafka-go) [![Go Report Card](https://goreportcard.com/badge/github.com/segmentio/kafka-go)](https://goreportcard.com/report/github.com/segmentio/kafka-go) [![GoDoc](https://godoc.org/github.com/segmentio/kafka-go?status.svg)](https://godoc.org/github.com/segmentio/kafka-go)
 
+#### Note:
+
+In order to better align  with our newly adopted Code of Contact, the kafka-go project is renaming our default branch to `main`.  
```

Comment:
Maybe say "has renamed" rather than "is renaming"? From the perspective of someone coming ti the project for information about the default branch changing, it may be more meaningful.